src/main/kotlin/com/sourcegraph/cody/commands/ui/CommandsTabPanel.kt

```diff
@@ -44,48 +50,75 @@ class CommandsTabPanel(
 
   init {
     layout = BoxLayout(this, BoxLayout.Y_AXIS)
+    setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10))
 
-    CommandId.values().forEach { command -> addCommandButton(command) }
+    fun addSeparator(name: String) {
+      val separator = TitledSeparator(name)
+      separator.maximumSize = Dimension(Int.MAX_VALUE, separator.preferredSize.height)
+      add(separator)
+    }
 
+    addSeparator("Edit Commands")
```

Comment:
this one should be included in the if-statement below

Reply:
Good point!